build-system/compile/sources.js

```diff
@@ -42,6 +42,9 @@ const COMMON_GLOBS = [
   'node_modules/intersection-observer/intersection-observer.install.js',
   'node_modules/promise-pjs/package.json',
   'node_modules/promise-pjs/promise.mjs',
+  'node_modules/rrule-without-luxon/dist/esm/src/**/*.js',
```

Comment:
I think you should be able to reference the import with `rrule-without-luxon` if you add the `package.json` here.